eventnotifier/slack/slack.go

```diff
@@ -180,16 +180,17 @@ func (n *Notifier) notifyForDisruptionCron(disruptionCron *v1beta1.DisruptionCro
 }
 
 func (n *Notifier) sendMessageToUserChannel(slackMsg slackMessage, logger *zap.SugaredLogger) error {
-	emailAddr, err := mail.ParseAddress(slackMsg.UserInfo.Username)
-	if err != nil {
-		logger.Infow("username could not be parsed as an email address", "err", err, "username", slackMsg.UserInfo.Username)
+	// emailAddr, err := mail.ParseAddress(slackMsg.UserInfo.Username)
```

Comment:
Hmm, there's a small problem with moving this check from `sendMessageToUserChannel` to `buildSlackMessage`. Now, in situations where we can't parse the username as an email address, we're going to be logging a warning rather than an info. We might need to double check here that UserEmail is defined and is a valid email address.